### PR TITLE
refurbishing of Payload Inspector

### PR DESCRIPTION
**Pull request by franzoni**
The Payload inspector (PI) is a facility accessible via the [AlCa/Db conditions browser](https://cms-conddb.cern.ch/cmsDbBrowser/) to visualize the content of payloads available in the conditions database; the PI discovers c++ classes in CMSSW and uses them to display the deserialized payload content in graphs customised by the calibration experts.
In this pull request (which does not affect CMS standard processing sequences):
   * general refurbishing of the c++ PI base classes
   * inclusion of new types of plots; calibration experts can inerit form them in their developments of customised views and charts to display conditions;
   * current types are : History, RunHistory, TimeHistory, Scatter, Histo1D and Histo2D
   * more details here in the [PI documentation twiki](https://twiki.cern.ch/twiki/bin/view/CMS/CmsDbBrowser#The_Payload_Inspector)

### CODE REVIEW

**Comment by franzoni** [comment]
please test

File: CondCore/Utilities/python/conddblib.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 __copyright__ = 'Copyright 2013, CERN'
 __credits__ = ['Giacomo Govi', 'Miguel Ojeda', 'Andreas Pfeiffer']
 __license__ = 'Unknown'
-__maintainer__ = 'Giacomo Govi'
+__maintainer__ = 'Miguel Ojeda'
```
Comment (mmusich) [inline CondCore/Utilities/python/conddblib.py]: @franzoni this seems to revert #17851 and #17278 from @ggovi. Is this intentional?

**Comment by franzoni** [comment]
hold

**Comment by franzoni** [comment]
superseded by https://github.com/cms-sw/cmssw/pull/18089

**Pull request closed without merging**